deploy/vos.yaml

```diff
@@ -19,6 +19,8 @@ vos:
     auth: "{{ fedcloud_sa }}"
   crystal_channelling_simulation.vo.egi.eu:
     auth: "{{ fedcloud_sa }}"
+  culturalheritage.vo.egi.eu:
+    auth: "{{ fedcloud_sa }}"
```

Comment:
this is missing, asking for membership now

Reply:
Approved